resources/views/cart/coupon.blade.php

```diff
@@ -2,20 +2,7 @@
     <div slot-scope="{ isOpen, toggle }">
         <coupon v-show="isOpen" v-slot="{ cart, removeCoupon, couponCode, inputEvents, applyCoupon, submitError }">
             <form class="text-sm" @submit.prevent="applyCoupon">
-                <div class="flex w-full flex-row gap-x-3">
-                    <x-rapidez-ct::input
-                        class="text-ct-primary w-60"
-                        name="couponCode"
-                        :placeholder="__('Enter code') . '...'"
-                        v-on="inputEvents"
-                        v-bind:value="couponCode"
-                        v-bind:disabled="$root.loading"
-                        required
-                    />
-                    <x-rapidez-ct::button.outline type="submit" loader>
-                        @lang('Apply')
-                    </x-rapidez-ct::button.outline>
-                </div>
+                @include('rapidez-ct::cart.partials.coupon-field')
```

Comment:
I think this input variation could be utilised best as a component, so classes and other attributes could be passed if necessary